build.sbt

```
@@ -186,3 +186,33 @@ webpack := {
 runner := runner.dependsOn(webpack).value
 dist := dist.dependsOn(webpack).value
 stage := stage.dependsOn(webpack).value
+
+// Docker
+
+enablePlugins(DockerPlugin)
```

```suggestion
enablePlugins(DockerPlugin)
```
should probably be made uniform with the other plugin dependencies